api/descriptions/custom-rules/custom-rules.md

```diff
@@ -1,2 +1,2 @@
-Custom runtime rules define precise runtime defense actions for containers, hosts, and Kubernetes audits.
+Defines a precise defense action for containers, hosts, and Kubernetes audits.
 Custom rules can be individually added to runtime and WAAS policies.
```

Comment:
@iansk : The text about WAAS policies is already there.

Reply:
@ssugandh Let's just change line 1 to this:

`Defines a precise runtime defense action for containers, hosts, Kubernetes audits, WAAS requests, and WAAS responses.`

And then go ahead and remove the second line.